study.md

```diff
@@ -38,29 +38,83 @@ As it makes sense to you please write the following in JSON format:
 - Has lived in two places recently: 123 Fake street, and 2 Muchinfo Road
 
 ```json
-<!-- your answer here -->
-```
-
-...I forgot, my roomate Dave has two goldfish, one named Bob (he's red) and the
-other named Mr.MagicNibbles (yellow) and bites a bit. Please copy the JSON you
-already wrote and include the information about Dave's fish.
-
-```json
-<!-- your answer here -->
-```
-
-## JSON Methods
-
-Using the JSON your wrote above, can you write a method on the JSON?
-
-```json
-<!-- your answer here -->
-```
-
-## JSON Comments
-
-Using the JSON your wrote above, can you write a comment in the JSON?
-
-```json
-<!-- your answer here -->
-```
+<!--
+{"Jason's Things"
+
+  "pet": "cat"
+  {
+    "type": "cat",
+    "name: "Mr Kitty",
+    "gender": "female",
+  }
+  "roommates":
+  {
+    "name":"Dave",
+    "name": "Miller"
+  }
+  "loves":
+  {
+    ["The Red Sox"], [" Patriots"], [" Bruins"]
```

Comment:
I was under the impression that each "object" that breaks down into new types of properties had to be broken up within the object that contained it all. Should it be more like this?: {"Jason's Things"
 +
 +  "pet": "cat"
 +  {
 +    "type": "cat",
 +    "name: "Mr Kitty",
 +    "gender": "female",
 +  }
 +  "roommates":
 +  {
 +    "name":"Dave",
 +    "name": "Miller"
 +  }
 +  "loves":
 +  {
 +    "The Red Sox", " Patriots", " Bruins"

I'm correct in breaking things like loves, pets and roommates into separate parts right?